homeassistant/components/playstation_network/image.py

```diff
@@ -70,31 +89,52 @@ async def async_setup_entry(
     async_add_entities(
         [
             PlaystationNetworkImageEntity(hass, coordinator, description)
-            for description in IMAGE_DESCRIPTIONS
+            for description in IMAGE_DESCRIPTIONS_ME + IMAGE_DESCRIPTIONS_ALL
         ]
     )
 
-
-class PlaystationNetworkImageEntity(PlaystationNetworkServiceEntity, ImageEntity):
+    for (
+        subentry_id,
+        friend_data_coordinator,
+    ) in config_entry.runtime_data.friends.items():
+        async_add_entities(
+            [
+                PlaystationNetworkFriendImageEntity(
+                    hass,
+                    friend_data_coordinator,
+                    description,
+                    config_entry.subentries[subentry_id],
+                )
+                for description in IMAGE_DESCRIPTIONS_ALL + IMAGE_DESCRIPTIONS_FRIENDS
+            ],
+            config_subentry_id=subentry_id,
+        )
```

Comment:
What would the benefit of having the avater of your friends as image entity? With steam they just get put as your entity_picture